packages/performance/src/resources/trace.ts

```diff
@@ -158,13 +158,13 @@ export class Trace implements PerformanceTrace {
 
   /**
    * Sets a custom metric to a specified value. Will create a new custom metric if one with the
-   * given name does not exist.
+   * given name does not exist. The value will be floored down to an integer.
```

Comment:
We need to do the same thing for incrementMetric method.

Reply:
Done.